README.md

```diff
@@ -65,7 +65,7 @@ First open a terminal and run the `yarn` command in the root directory to get th
 
 4. Synchronize your [android](/android) project with Gradle if opened in Android Studio.
 
-> **Warning**  
+> **Warning**
```

Comment:
Please update these to the current GitHub format (see [docs](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#alerts)):
```markdown
> [!WARNING]
> ...
```

Reply:
Thanks, fixed.